.azure/gpu-tests.yml

```diff
@@ -18,12 +18,12 @@ jobs:
     # how much time to give 'run always even if cancelled tasks' before stopping them
     cancelTimeoutInMinutes: "2"
 
-    pool: azure-gpus-spot
+    pool: lit-rtx-3090
 
     container:
       image: "pytorchlightning/pytorch_lightning:base-cuda-py3.9-torch1.11-cuda11.3.1"
       #endpoint: azureContainerRegistryConnection
-      options: "--runtime=nvidia -e NVIDIA_VISIBLE_DEVICES=all --shm-size=32g --name ci-container -v /usr/bin/docker:/tmp/docker:ro"
+      options: "--gpus=all --shm-size=16g --name ci-container -v /usr/bin/docker:/tmp/docker:ro"
```

Comment:
isn't it also needed to have a step `set visible devices` as in main repo?

Reply:
good catch